persistence/schema/account.go

```diff
@@ -0,0 +1,28 @@
+package schema
+
+const (
+	AccountTableName   = "Account"
+	AccountTableSchema = `(
+			id         BIGINT PRIMARY KEY,
+			address    STRING NOT NULL UNIQUE,
+		)`
+	AccountMetaTableName   = "Account_meta"
```

Comment:
> Having a metadata table implies it is metadata - not used for business logic, which is not the case here. Why split it out?
To not to have to duplicate the address field everytime you edit the balance
> We just started storing two BIGINTs id and account_id, and added in a foreign constraint w/ 2 tables. This can be made much simpler.
How so? Please leave actionables
> This is missing created_at and updated_at per the spec
Added created_at - updated_at is not needed